src/lib/database.ts

```diff
@@ -210,15 +210,15 @@ export class Database {
 					tmp = new Sequelize(null, null, null, opts)
 				}
 			} catch (e) {
-				return reject(e)
+				reject(e)
```

Comment:
`return` keyword is important here, it will throw an error if we enter the `catch` section (`tmp` is null and L215 we call `tmp.query`)
```suggestion
				return reject(e)
```